java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -229,14 +198,14 @@ private PreparedStatement generatePreparedStatement(
                     stmt.setBigDecimal(placeholderIdx++, (java.math.BigDecimal) row.get(i));
                     break;
                 case INTERVAL:
-                    if (targetDbType == DatabaseType.POSTGRES) {
+                    if (jdbcDialect instanceof PostgresDialect) {
```

Comment:
Can we introduce the abstraction of processing interval and jsonb in the `JdbcDialect` interface so that we don't need to do a special hack here.